Articles/Blog/2022-03-sc21-rse-workshop.md

```diff
@@ -0,0 +1,204 @@
+for heterogeneous architectures and next-generation science, and
+coordinating a wide range of software and teams, with all the resulting
+technical and sociological challenges. In response, ECP has developed an
+open, hierarchical software ecosystem, building a "team of teams" based on
+community collaboration and shared policies, while advancing scientific
+productivity through better scientific software. She concluded her talk with
+a shout-out to her ECP RSE colleagues at Argonne and elsewhere, describing
+them as "the linchpin of all things exascale - and beyond"!
+
+### Short talks
+
+Marion Weinzierl gave the first short talk on "Training HPC RSEs: What we
+learned from our performance." This was based on training workshops that
+they gave, with seven sessions over seven months. These lessons were
+tutorial-style in the mornings and hackathon-style in the afternoons. They
+were taken by 79 people in 13 teams, with 20-50 participants per session.
+Lessons from this work included that 1) HPC research software developers
+need a sound foundation in in-depth computer science and hardware-ware
+programming to be able to understand performance analysis and turn insights
+into performance optimization; 2) performance analysis has to be a team
+effort and you (really) need to know the code base; 3) performance analysis
+has to be a first-class activity for high-performance code development; and
+4) training for HPC-RSEs has to take into account the range of people who
+develop HPC software, who have different backgrounds and skills, interests,
+and priorities, and that it
+needs to spell out the basic concepts that are used and needs to bridge the
+gap between hardware and software knowledge. They also have written a
+detailed report: <https://doi.org/10.5281/zenodo.5155502>.
+
+The next short talk was on "Building an AI-ready RSE Workforce," by Ying
+Zhang. This talk discussed AI's increasing impact on research software, and
+what the University of Florida is doing to give RSEs (and others) AI skills,
+including an AI curriculum with a certificate in AI, as well as AI training.
+
+The final short talk, "An Exploration of the Mentorship Needs of Research
+Software Engineers" was presented by Reed Milewicz. The talk discussed
+mentorship as a strategy to enable the career growth and retention of RSEs;
+key needs that RSEs may have in providing and receiving mentorship; and
+directions for future work. The talk defined mentorship as a relationship in
+which a more experienced or more knowledgeable person (a mentor) helps to
+guide a less experienced or less knowledgeable person (a mentee), and talked
+about the benefits for both the mentee and mentor. Milewicz found that RSEs
+need interdisciplinary mentorship networks, long-term mentoring
+relationships, and training in soft skills. The talk called for us to start
+mentorships now, to build explicit institutional support for them, and to
```

Comment:
```suggestion
mentorships now, to build explicit institutional support for these mentorships, and to
```

Reply:
I think it is okay as is.  We already have the word twice in the same sentence.